### ISSUE TITLE
Consider building packages for Rocky Linux

### ISSUE THREAD

**Issue by fraenki** [openzfs/zfs#11846]
### Describe the feature would like to see added to OpenZFS
CentOS 8 is scheduled to be EOL'ed [on 2021-12-31](https://blog.centos.org/2020/12/future-is-centos-stream/). [Rocky Linux](https://github.com/rocky-linux/rocky/) aims to be a compatible replacement. As such it would be nice to have an official OpenZFS repository available for it.

First Rocky Linux BETA is [scheduled for 2021-04-30](https://rockylinux.org/news/community-update-march-2021/).

### How will this feature improve OpenZFS?

OpenZFS provides packages for CentOS 8, this will no longer be necessary after 2021-12-31. Since Rocky Linux is the community effort to continue CentOS 8, it probably makes sense to build packages for it.

### Additional context

Providing packages for Rocky Linux would allow former users of CentOS 8 to switch to Rocky Linux while keeping OpenZFS.
I'm one of them. :) Other projects have also announced their own CentOS 8 rebuilds, but Rocky Linux seems to be the most popular choice.

**Comment by mskarbek** [comment]
If any then the most reasonable would be using RHEL for stable release and CentOS Stream to be prepared for what comes next. If Rocky Linux deliver as a compatible replacement, then there is no reason it couldn't use RHEL repo. That way OpenZFS will cover all RHEL forks like Rocky or Alma.

**Comment by behlendorf** [comment]
Our intention is to continue building packages for RHEL 8.x releases even after CentOS 8 is EOL.  Based on Rocky Linux's goal to be a binary compatible replacement our expectation is these packages will be compatible.

**Comment by devZer0** [comment]
Yes, if RockyLinux aims for RHEL compatibility, there is no need for a specialized RockyLinux Build

You can see at https://openzfs.github.io/openzfs-docs/Getting%20Started/RHEL%20and%20CentOS.html  that ZFS for Centos and RHEL is the same package
